Gruntfile.js

```diff
@@ -8,14 +8,23 @@ module.exports = function (grunt) {
 
   // Node and client side JS have slightly different JSHint directives
   // We'll create 2 versions with .jshintrc as a baseline
-  var browserJSHint = grunt.file.readJSON('.jshintrc');
-  var nodeJSHint = grunt.file.readJSON('.jshintrc');
+  var browserJSHint = grunt.file.readJSON('node_modules/mofo-style/linters/.jshintrc');
+  var nodeJSHint = grunt.file.readJSON('node_modules/mofo-style/linters/.jshintrc');
```

Comment:
You actually only need to load it once now since the new jshintrc has browser and node environments set.